src/code/z_actor.c

```diff
@@ -457,439 +1246,3472 @@ f32 Actor_XZDistanceToPoint(Actor* actor, Vec3f* point) {
+                         spA4);
+        // temp_s1_11 = phi_s1_2;
+
+        spAC = phi_s1_2;
+
+        // spAC = temp_s1_11;
+        func_800B9E84(&spAC, (s32)globalCtx->actorCtx.unk4);
+        // temp_s1_11->words.w0 = 0xE7000000;
+        // temp_s1_11->words.w1 = 0;
+        // temp_s1_12 = temp_s1_11 + 8;
+        gDPPipeSync(spAC++);
+
+        // temp_s1_12->words.w1 = -0xF8;
+        // temp_s1_12->words.w0 = 0xF9000000;
+        // temp_s1_13 = temp_s1_12 + 8;
+        gDPSetBlendColor(spAC++, 255, 255, 255, 8);
+
+        // temp_s1_14 = temp_s1_13 + 8;
+        // temp_s1_13->words.w0 = ((sp34->unk_B4C - 1) & 0xFFF) | 0xFF100000;
+        // temp_s1_13->words.w1 = sp34->unk_B5C;
+        gDPSetColorImage(spAC++, G_IM_FMT_RGBA, G_IM_SIZ_16b, ((globalCtx->pauseBgPreRender.width - 1) & 0xFFF),
+                         globalCtx->pauseBgPreRender.fbuf);
+
+        // temp_a1 = &spAC;
+        // spAC = temp_s1_14;
+        // func_8016FDB8(&globalCtx->pauseBgPreRender, temp_a1, (void* ) spA4, spA8, 1U);
+        func_8016FDB8(&globalCtx->pauseBgPreRender, &spAC, (void*)spA4, __gfxCtx->zbuffer, 1U);
+        //
+        // POLY_OPA_DISP = temp_s1_14;
+        POLY_OPA_DISP = spAC;
+    }
+
+    // temp_s1_15 = OVERLAY_DISP;
+    // temp_s1_15->words.w0 = 0xE7000000;
+    // temp_s1_15->words.w1 = 0;
+    // temp_s1_16 = temp_s1_15 + 8;
+    spAC = OVERLAY_DISP;
+    gDPPipeSync(spAC++);
+
+    // temp_s1_16->words.w0 = 0xEF002C30;
+    // temp_s1_16->words.w1 = 0x00504345;
+    // temp_s1_17 = temp_s1_16 + 8;
+    gDPSetOtherMode(spAC++,
+                    G_AD_DISABLE | G_CD_MAGICSQ | G_CK_NONE | G_TC_FILT | G_TF_BILERP | G_TT_NONE | G_TL_TILE |
+                        G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
+                    G_AC_THRESHOLD | G_ZS_PRIM | G_RM_CLD_SURF | G_RM_CLD_SURF2);
+
+    // temp_s1_17->words.w0 = 0xFC61E6C3;
+    // temp_s1_17->words.w1 = 0x11CF9FCF;
+    // temp_s1_18 = temp_s1_17 + 8;
+    gDPSetCombineLERP(spAC++, 1, TEXEL0, PRIMITIVE, 0, 1, TEXEL0, PRIMITIVE, 0, 1, TEXEL0, PRIMITIVE, 0, 1, TEXEL0,
+                      PRIMITIVE, 0);
+
+    // temp_s1_18->words.w0 = 0xFA000000;
+    // temp_s1_18->words.w1 = 0x4A00004A;
+    // spAC = temp_s1_18 + 8;
+    gDPSetPrimColor(spAC++, 0, 0, 74, 0, 0, 74);
+
+    func_800B9E84(&spAC, (s32)globalCtx->actorCtx.unk4);
+
+    OVERLAY_DISP = spAC;
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+#else
+void func_800B9EF4(GlobalContext* globalCtx, s32 numActors, Actor** actors);
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800B9EF4.s")
+#endif
+
+s32 func_800BA2D8(GlobalContext* globalCtx, Actor* actor) {
+    return func_800BA2FC(globalCtx, actor, &actor->projectedPos, actor->projectedW);
+}
+
+s32 func_800BA2FC(GlobalContext* globalCtx, Actor* actor, Vec3f* arg2, f32 arg3) {
+    if ((-actor->uncullZoneScale < arg2->z) && (arg2->z < (actor->uncullZoneForward + actor->uncullZoneScale))) {
+        f32 temp_f14;
```

Comment:
If I remember correctly, this temp_f14 wasn't needed. I just forgot to remove it from the decompme scratch.